src/pages/en/guides/migrate-to-astro/from-gatsby.mdx

```diff
@@ -2,67 +2,576 @@
+
+- **@astrojs/react**: to reuse some existing React UI components in your new Astro site, or keep writing with React components.
+
+- **@astrojs/image**: to replace Gatsby's Image plugin with Astro's own image-optimizing components. (Experimental: works in `.astro` and `.mdx` files only.)
+
+- **@astrojs/mdx**: to bring existing MDX files from your Gatsby project, or to use MDX in your new Astro site.
+
+### Repurpose config files
+
+Gatsby has several top-level configuration files that also include site and page metadata and are used for routing. You will not use any of these `gatsby-*.js` files in your Astro project, but there may be some content that you can reuse:
+
+- `gatsby-config.js`: Move your `siteMetadata: {}` into `src/data/siteMetadata.js` (or `siteMetadata.json`) to import data about your site (title, description, social accounts etc.) into page layouts.
+
+- `gatsby-browser.js`: Consider adding anything used here directly into your main layout's `<head>` tag.
+
+- `gatsby-node.js`: You will not need to create your own nodes in Astro, but viewing the schema in this file may help you with defining types in your Astro project.
+
+- `gatsby-ssr.js`: If you choose to use SSR in Astro, you will add and configure the adapter of your choice directly in `astro.config.mjs`.
+
+
+### Put your code in `src`
+
+1. **Delete** Gatsby's `public/` folder. 
+    
+    Gatsby uses the `public/` directory for its build output, so you can safely discard this folder. You will no longer need a built version of your Gatsby site. (Astro uses `dist/` by default for the build output.)
+
+2. **Rename** Gatsby's `static/` folder to `public/`, and use it as Astro's `public/` folder. 
+
+    Astro uses a folder called `public/` for static assets. You can alternatively copy the contents of `static/` into your existing Astro `public/` folder.
+
+3. **Copy or Move** Gatsby's other files and folders (e.g. `components`, `pages` etc.) as needed into your Astro `src/` folder as you rebuild your site, following [Astro's project structure](/en/core-concepts/project-structure/).
+
+  Astro's `src/pages/` folder is a special folder used for file-based routing to create your site's pages and posts from `.astro`, `.md` and `.mdx` files. You will not have to configure any routing behavior for your Astro, Markdown and MDX files.
+
+  All other folders are optional, and you can organize the contents of your `src/` folder any way you like. Other common folders in Astro projects include `src/layouts/`, `src/components`, `src/styles`, `src/scripts`.
+
+
+### Convert JSX files to `.astro` files
+
+Here are some tips for converting a Gatsby `.js` component into a `.astro` component:
+
+1. Use only the `return()` of the existing Gatsby component function as your HTML template.
+
+1. Change any [Gatsby or JSX syntax to Astro](#reference-convert-to-astro-syntax) (e.g. `<Link to="">`, `{children}`, `className`, inline style objects) or to HTML web standards.
```

Comment:
The mid-sentence parentheses threw me while reading. What do you think of this? I’ve also cut the mention of inline style objects as these _do_ work in Astro even if they’re less commonly used by Astro users.

```suggestion
1. Change any [Gatsby or JSX syntax to Astro syntax](#reference-convert-to-astro-syntax) or to HTML web standards. This includes `<Link to="">`, `{children}`, and `className`, for example.
```